README.md

```diff
@@ -1,360 +1,81 @@
-});
-
-// a firstname & company
-index.search('jimmie paint', function(err, content) {
-  console.log(content.hits);
-});
-```
-
-### Configure
-
-Settings can be customized to tune the search behavior. For example, you can add a custom sort by number of followers to the already great built-in relevance:
-
-> for this use: [**admin** API key](https://www.algolia.com/doc/guides/security/api-keys/#admin-api-key) in client instantiation
-
-```js
-index.setSettings({
-  'customRanking': ['desc(followers)']
-}, function(err, content) {
-  console.log(content);
-});
-```
-
-You can also configure the list of attributes you want to index by order of importance (first = most important):
-
-**Note:** Since the engine is designed to suggest results as you type, you'll generally search by prefix.
-In this case the order of attributes is very important to decide which hit is the best:
-
-```js
-index.setSettings({
-  'searchableAttributes': [
-    'lastname',
-    'firstname',
-    'company',
-    'email',
-    'city',
-    'address'
-  ]
-}, function(err, content) {
-  console.log(content);
-});
+<p align="center">
+  <a href="https://www.algolia.com">
+    <img alt="Algolia for JavaScript" src="https://raw.githubusercontent.com/algolia/algoliasearch-client-common/master/banners/javascript.png" >
+  </a>
+
+  <h4 align="center">The perfect starting point to integrate <a href="https://algolia.com" target="_blank">Algolia</a> within your JavaScript project</h4>
+
+  <p align="center">
+    <a href="https://travis-ci.org/algolia/algoliasearch-client-javascript"><img src="https://img.shields.io/travis/algolia/algoliasearch-client-javascript/master.svg" alt="Build Status"></img></a>
+    <a href="https://npmjs.org/package/algoliasearch"><img src="https://img.shields.io/npm/v/algoliasearch.svg?style=flat-square" alt="NPM version"></img></a>
+    <a href="http://npm-stat.com/charts.html?package=algoliasearch"><img src="https://img.shields.io/npm/dm/algoliasearch.svg?style=flat-square" alt="NPM downloads"></a>
+    <a href="https://www.jsdelivr.com/package/npm/algoliasearch"><img src="https://data.jsdelivr.com/v1/package/npm/algoliasearch/badge" alt="jsDelivr Downloads"></img></a>
+    <a href="LICENSE.txt"><img src="https://img.shields.io/badge/license-MIT-green.svg?style=flat-square" alt="License"></a>
+  </p>
+</p>
+
+<p align="center">
+  <a href="https://www.algolia.com/doc/api-client/getting-started/install/javascript/" target="_blank">Documentation</a>  •
+  <a href="https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/js/" target="_blank">InstantSearch</a>  •
+  <a href="https://discourse.algolia.com" target="_blank">Community Forum</a>  •
+  <a href="http://stackoverflow.com/questions/tagged/algolia" target="_blank">Stack Overflow</a>  •
+  <a href="https://github.com/algolia/algoliasearch-client-javascript/issues" target="_blank">Report a bug</a>  •
+  <a href="https://www.algolia.com/support" target="_blank">Support</a>
```

Comment:
Do we add this link to other API clients' readmes as well? Asking because we don't necessarily provide the same support to everybody.